source/quick_start_guide/index.html.md.erb

```diff
@@ -25,11 +25,20 @@ The following are some examples of base URLs:
 
 <div style="height:1px;font-size:1px;">&nbsp;</div>
 
-### Choose an endpoint  
+### Append the current version of the API
+The current version of the API is **v1**.
+
+You should add the version of the API you want to use to the end of the base URL. This step is optional for version 1 of the API and will become required when version 1 is no longer supported.
+
+For example, the full Base URL for version 1 of the `local-authority-eng` register is `https://local-authority-eng.register.gov.uk/v1/`.
```

Comment:
Is it a expected that `https://local-authority-eng.register.gov.uk/v1` returns a 404?

Reply:
Nope, that's not ideal. I'll raise a PR to fix that.